framework/doc/content/source/postprocessors/DiscreteVariableResidualNorm.md

```diff
@@ -0,0 +1,50 @@
+# DiscreteVariableResidualNorm
+
+This postprocessor computes one of the following discrete norms of the nonlinear residual
+vector $\mathbf{r}$ for a variable $u$ over a chosen domain $\Omega$:
+
+- $\ell_1$:
+
+  !equation
+  \|r\|_1 = \sum\limits_{i\in \mathcal{I}_{u,\Omega}} |r_i|
+
+- $\ell_2$:
+
+  !equation
+  \|r\|_2 = \sqrt{\sum\limits_{i\in \mathcal{I}_{u,\Omega}} |r_i|^2}
+
+- $\ell_\infty$:
+
+  !equation
+  \|r\|_\infty = \max\limits_{i\in \mathcal{I}_{u,\Omega}} |r_i|
+
+where $\mathcal{I}_{u,\Omega}$ is the set of residual vector indices corresponding to the
+solution variable $u$ and the domain $\Omega$.
+
+Note that some of these norms have a mesh-size bias. For example, if all vector entries
```

Comment:
```suggestion
Note that because this is not a spatial integral, but rather an algebraic vector norm, some of these norms have a mesh-size bias. For example, if all vector entries
```

Reply:
I probably wouldn't say this since the continuous norms have a different mesh-related bias - they're biased on the domain size rather than the number of elements.

Reply:
ok edited. There are no perfect norms